Task flow compare

cmajel opened the pull request:
Adding documentation notes on how we came to our MVP task flow for our first usability test, and how it compares with the Christmas tree permitting task flow.

[Preview of this page](https://github.com/USDAForestService/USFS-timber-permitting/blob/task-flow-compare/docs/design-deliverables/online-timber-permitting-ui/2019-12--mvp-usability-test/mvp-firewood-task-flow.md)

cc @tram and @MelissaBraxton for thoughts!

MelissaBraxton commented:
This is a great start @cmajel. A few thoughts:
-  I'm concerned that lining up the firewood task flow against the Christmas tree flow in this way makes it appear as though the firewood permitting flow is more set than it actually is. (I know your text tries to make it clear that there's more to be learned, but I'm not sure it comes through strong enough in how this is laid out. Can we make it more clear that this is looking at the delta between Christmas trees and a first iteration, and that we have ideas for how we'd revise I1 to get to I2? 
 - for item 5. in the flow (Login) Is there something we can say about what Qs we have or hypotheses we think would be important to test in the next round of research? I.e, make it more specific.  
 - I'd also suggest explaining a bit what we mean by MVP--the assumptions we're making for how and where this will be piloted and how that guided the prototype. Maybe just put a placeholder link to more info, since I'll be adding the roadmap/risks/assumptions content in the next few days. Once we move beyond the MVP, the flow will likely grow to include processes required for a fraud resistant permit/tag. We want to make sure folks looking at this know that we're aware of and planning for that future. 
 - I think the last user story in the list ("infirmed permittee") was deprioritized for I1--let's remove it.

cmajel commented:
@MelissaBraxton thanks for the great feedback! 

I agree that we should think about how this fits in with our overall documentation and what should live where. My primary goal here was to document where and why we chose to diverge from the Christmas tree permit design. I suspect a lot of this (perhaps everything before the `Comparing...`section) overlaps with the upcoming product roadmap documentation and maybe should live somewhere else.

I took a stab at updating it while we're writing the additional pieces. Changes include:

> I'm concerned that lining up the firewood task flow against the Christmas tree flow in this way makes it appear as though the firewood permitting flow is more set than it actually is. (I know your text tries to make it clear that there's more to be learned, but I'm not sure it comes through strong enough in how this is laid out. Can we make it more clear that this is looking at the delta between Christmas trees and a first iteration, and that we have ideas for how we'd revise I1 to get to I2?

I added more qualifiers that this is for our Dec 2019 MVP, not a final product. Regarding your last sentence, would a link back to the research read out satisfy this?

> for item 5. in the flow (Login) Is there something we can say about what Qs we have or hypotheses we think would be important to test in the next round of research? I.e, make it more specific.

Added!

> I'd also suggest explaining a bit what we mean by MVP--the assumptions we're making for how and where this will be piloted and how that guided the prototype. Maybe just put a placeholder link to more info, since I'll be adding the roadmap/risks/assumptions content in the next few days. Once we move beyond the MVP, the flow will likely grow to include processes required for a fraud resistant permit/tag. We want to make sure folks looking at this know that we're aware of and planning for that future.

I took a stab at this at the top of the doc, but again, maybe all this exists somewhere else and this doc becomes purely an addendum to the usability test readout, and gives notes on why we diverged from Christmas tree permits. 

> I think the last user story in the list ("infirmed permittee") was deprioritized for I1--let's remove it.

I had these under the `two people voted for it` category, but I can remove it (my list may be out of date)
